eth/handler.go

```diff
@@ -43,6 +44,14 @@ import (
 const (
 	softResponseLimit = 2 * 1024 * 1024 // Target maximum size of returned blocks, headers or node data.
 	estHeaderRlpSize  = 500             // Approximate size of an RLP encoded block header
+
+	// txChanSize is the size of channel listening to NewTxsEvent.
+	// The number is referenced from the size of tx pool.
+	txChanSize = 4096
+)
+
+var (
+	forkChallengeTimeout = 15 * time.Second // Time allowance for a node to reply to the DAO handshake challenge
```

Comment:
This is not a new idea (it used to just be hardcoded in the early handler `handle()` fn requiredHash check piece, just now set to more visible global var. _However_, it used to be __5 seconds__. And you're right, 15 seconds does seem a little high. Open idea.